src/excmds.ts

```diff
@@ -3281,30 +3281,61 @@ export async function yankimage(url: string): Promise<void> {
 
 /** Change active tab.
 
-    @param index
-        Starts at 1. 0 refers to last tab of the current window, -1 to penultimate tab, etc.
+    @param id
+        A string following the following format: "[0-9]+.[0-9]+", the first number being the index of the window that should be selected and the second one being the index of the tab within that window.
+
+        A bare number means the current window is used.
 
         "#" means the tab that was last accessed in this window
 
-    This is different from [[taball]] because `index` is the position of the tab in the current window.
+        A non integer string means to search the URL and title for matches
```

Comment:
"Any other string searches for matches and switches to the first one."

I think I would expect `:tab` to only switch to matches in the current window and `:taball` to switch to matches in any window.